docs/auth/example-lh-auth.test.js

```diff
@@ -0,0 +1,134 @@
+/** @typedef {import('./node_modules/lighthouse/types/lhr')} LH */
+
+const puppeteer = require('puppeteer');
+const lighthouse = require('lighthouse');
+const server = require('./server/server.js');
+
+const DEBUG_PORT = 8042;
```

Comment:
feels a bit weird to call this debug port from a typical test author's perspective, wdyt about

- `BROWSER_PORT`
- `CHROME_PORT`
- `CHROME_DEBUG_PORT`

?